src/drivers/uart/src/lib.rs

```diff
@@ -15,3 +15,8 @@ pub mod opentitan;
 pub mod pl011;
 #[cfg(feature = "sifive")]
 pub mod sifive;
+
+/* Calculate divisor. Do not floor but round to nearest integer. */
+pub fn uart_baudrate_divisor(baudrate: usize, refclk: usize, oversample: usize) -> usize {
```

Comment:
This function looks like it never gets used.

Reply:
It doesn't. It is also seems unnecessary because the maximum baudrate is used when possible.